staging/src/k8s.io/apiserver/pkg/storageversion/manager.go

```diff
@@ -90,6 +91,9 @@ func (s *defaultManager) AddResourceInfo(resources ...*ResourceInfo) {
 	s.mu.Lock()
 	defer s.mu.Unlock()
 	for _, r := range resources {
+		if r == nil {
```

Comment:
this seems like a bug.  how did we get a nil?

Reply:
We didn't. Had the check just to be safe. I can remove it